src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -635,8 +636,10 @@ function prepareQuestionForm(type) {
 
         $('#contribForm').show();
         fixContribQnGiverRecipient(NEW_QUESTION);
-        setDefaultContribQnVisibility(NEW_QUESTION);
         setContribQnVisibilityFormat(NEW_QUESTION);
+        if (!didCopyOptions) {
```

Comment:
I found it necessary as `setDefaultContribQnVisibility` should be called from here, but there's no way to determine if the options were copied from here.